src/NuGet.Clients/NuGet.Tools/NuGet.Tools.csproj

```
@@ -127,6 +128,9 @@
     </ProjectReference>
   </ItemGroup>
   <ItemGroup>
+    <PackageReference Include="Microsoft.Internal.VisualStudio.Shell.Embeddable" ExcludeAssets="build">
+      <Version>16.3.28923.271</Version>
+    </PackageReference>
```

Sometimes, MSBuild has a hard time reading nested properties

```suggestion
<PackageReference Include="Microsoft.Internal.VisualStudio.Shell.Embeddable" ExcludeAssets="build" Version="16.3.28923.271" />
```